components/UploadedFilePreview.js

```diff
@@ -137,7 +137,7 @@ const UploadedFilePreview = ({
     );
   } else {
     const resizeWidth = Array.isArray(size) ? max(size) : size;
-    const img = <img src={imagePreview(url, null, { width: resizeWidth })} alt={alt || fileName} />;
+    const img = <img src={imagePreview(url, null, { width: ['100%', resizeWidth] })} alt={alt || fileName} />;
```

Comment:
@Betree : Yeah, you are right, thinking more about this I think there's a simpler fix for this also preserving the original design; where we have a larger image for the mobile. I've updated the original screen-cast with the new one. 😄 Let me know how it goes.

Reply:
I also feel we can handle this better maybe rather than the same function trying to handle all different file types and rendering the preview images in different ways, but I hope this fixes the original issue now; maybe we should refactor this part of code. 🤔 🤔